### PR TITLE
2.9 release updates

### PR DESCRIPTION
**Pull request by kmova**
- Updated docs to reflect new engines that have been added recently
- Updated Alpha features
- References to 2.9.0 and links to previous doc snapshots
- Update the minimum required K8s to 1.18
- Replaced references to fields in the sample outputs with `...`, to avoid the need to update the version numbers on every release

### CODE REVIEW

File: docs/overview.md [hunk 1]
```diff
@@ -7,117 +7,101 @@ sidebar_label: Overview
...
+- Being Portable across all Kubernetes distributions - On-premise and Cloud.
+- Increased Developer and Platform SRE Productivity.
+- Easy to use compared to other solutions.
+- Excellent Community Support.
+- Open Source and Free.
 
-Check out what users of OpenEBS have to say about their experience in <a href="https://github.com/openebs/openebs/blob/master/ADOPTERS.md" target="_blank">OpenEBS Adoption stories</a>.
 
-## Types of OpenEBS Storage Engines
+## Local Volumes
```
Comment (ranjithwingrider) [inline docs/overview.md:53]: Is it good to mention its volume binding mode as `WaitForFirstConsumer`? This parameter says all for a Kubernetes user.

File: docs/overview.md [hunk 2]
```diff
@@ -7,117 +7,101 @@ sidebar_label: Overview
...
-### Selecting the right storage engine
+### Replicated Volume Quickstart Guides
+
+Installing OpenEBS in your cluster is as simple as running a few `kubectl` or `helm` commands. Here are the list of our Quickstart guides with detailed instructions for each storage engine.
+
+- [Mayastor](/docs/next/ugmayastor.html)
+- [cStor](https://github.com/openebs/cstor-operators/blob/master/docs/quick.md)
+- [Jiva](https://github.com/openebs/jiva-operator)
+
+## Selecting the right storage engine
```
Comment (ranjithwingrider) [inline docs/overview.md:85]: Are we not mentioning backup and restore?

File: docs/quickstart.md [hunk 3]
```diff
@@ -19,29 +19,18 @@ sidebar_label: Quickstart
```
Comment (ranjithwingrider) [inline docs/quickstart.md:19]: We need to update the OpenEBS configuration sequence diagram with MayaStor engine. We can add it to backlog.

File: docs/quickstart.md [hunk 4]
```diff
@@ -19,29 +19,18 @@ sidebar_label: Quickstart
 
 <div class="emphasize">
     <ul>
-        <li>Kubernetes 1.13+ installed. Latest tested Kubernetes version is 1.17.2.</li>
-        <li>For using features like Local PV and Backup & Restore, you must have Kubernetes version 1.13 or above.
+        <li>Based on the storage requirements for your stateful workloads, <a href="/docs/next/casengines.html#when-to-choose-which-cas-engine"> select the OpenEBS Storage engines to be installed</a>.
```
Comment (ranjithwingrider) [inline docs/quickstart.md:22]: The linked page should include MayaStor as well right?

File: docs/releases.md [hunk 5]
```diff
@@ -6,10 +6,122 @@ sidebar_label: Releases
 
 ------
 
-## 2.7.0 - Mar 16 2021
+## 2.9.0 - May 15 2021
 
 <br><font size="4">Latest Release</font><br/> (Recommended)<br/>
 
+OpenEBS v2.9 is another maintenance release before moving towards 3.0 primarily focusing on enhancing the E2e tests and build/release workflows. This release includes fixes for user-reported critical bugs as well as fixes and enhancements to improve the E2e test coverage. There has been some significant progress made on the alpha features as well.
```
Comment (ranjithwingrider) [inline docs/releases.md:13]: Why E2e is like this way? better to go with E2E?

File: docs/releases.md [hunk 6]
```diff
@@ -6,10 +6,122 @@ sidebar_label: Releases
...
+
+- Enhanced [ZFS Local PV](https://github.com/openebs/zfs-localpv) to use a custom node label called `openebs.io/nodeid` to set the node affinity for the provisioned volume. By default, the value will be the same as `kubernetes.io/hostname`.  Using a custom label like this will help in quickly migrating the volumes to a new node in cases where a node fails and the user needs to move the underlying disks to a new node in the cluster. After moving the disks, the user can set the `openebs.io/nodeid` with the value used in the previous node. (https://github.com/openebs/zfs-localpv/issues/304). You can read more about this feature [here](https://github.com/openebs/zfs-localpv/blob/master/docs/faq.md#8-how-to-migrate-pvs-to-the-new-node-in-case-old-node-is-not-accessible).
+- Enhanced [cStor Velero plugin](https://github.com/openebs/velero-plugin/pull/154) to allow users to specify a custom timeout for completing snapshot operations. The time out can be configured via the `restApiTimeout ` field in VolumeSnapshotLocal. See [example](https://github.com/openebs/velero-plugin/blob/master/example/06-volumesnapshotlocation.yaml). (https://github.com/openebs/velero-plugin/issues/148)
```
Comment (ranjithwingrider) [inline docs/releases.md:19]: timeout or time out?

Comment (ranjithwingrider) [inline docs/releases.md:19]: VolumeSnapshotLocal  -> VolumeSnapshotLocation?

File: docs/releases.md [hunk 7]
```diff
@@ -6,10 +6,122 @@ sidebar_label: Releases
...
+- Added helm chart for [LVM Local PV](https://github.com/openebs/lvm-localpv/tree/master/deploy/helm/charts). 
+- Enhanced [LVM Local PV] to allow users to specify a pattern string of volume groups from which LVM Local PV should be provisioned. This feature will help in cases where a node can have multiple volume groups or volume group names across the cluster have to be unique. (https://github.com/openebs/lvm-localpv/pull/28)
```
Comment (ranjithwingrider) [inline docs/releases.md:21]: [LVM Local PV]  -> Hyperlink is missing?

File: hack/cspell-ignore.txt [hunk 8]
```diff
@@ -10,6 +10,7 @@ Rsvd
 WSREP
```
Comment (ranjithwingrider) [inline hack/cspell-ignore.txt:10]: Should we add E2E or e2e or E2e? I have seen these 3 terms.

File: hack/cspell-words.txt [hunk 9]
```diff
@@ -388,6 +393,7 @@ sendtargets
 serviceaccount
 serviceaccounts
 sgcluster
+sharded
```
Comment (ranjithwingrider) [inline hack/cspell-words.txt:396]: Do we have any word sharded or it is shared? I couldn't find sharded.

**Review by ranjithwingrider — COMMENTED** [review]
@kmova Please find my suggestions.

File: docs/alphafeatures.md [hunk 10]
```diff
@@ -34,11 +35,19 @@ For additional details and detailed instructions on how to get started with Open
 
 <hr>
 
-## Dynamic Local PV - LVM
+## OpenEBS CLI
 
-OpenEBS is developing a CSI driver for provisioning Local PVs that are backed by LVM. 
+OpenEBS is developing a kubectl plugin for openebs called `openebsctl` that can help perform administrative tasks on OpenEBS volumes and pools.
```
[the remark this answers is not in the document]
Comment (kmova) [reply docs/alphafeatures.md:40]: The existing sentence is fine.

File: docs/installation.md [hunk 11]
```diff
@@ -420,7 +420,7 @@ As a next step [verify](#verifying-openebs-installation) your installation and d
```
[the remark this answers is not in the document]
Comment (kmova) [reply docs/installation.md:420]: yes. Will take this up in the next PR that improves the overall user guides.

File: docs/overview.md (hunk 1 above)
Comment (kmova) [reply docs/overview.md:53]: we support both. Will take this up after some reviews through slack.

File: docs/overview.md (hunk 2 above)
Comment (kmova) [reply docs/overview.md:85]: yeah.. removed it for now. To avoid confusion.

File: docs/quickstart.md (hunk 3 above)
Comment (kmova) [reply docs/quickstart.md:19]: yes. coming up in new PR.

File: docs/quickstart.md (hunk 4 above)
Comment (kmova) [reply docs/quickstart.md:22]: yes. next PR.

File: docs/releases.md (hunk 5 above)
Comment (kmova) [reply docs/releases.md:13]: all are valid. But we can probably take this uniformity across all in one PR.

File: docs/releases.md (hunk 6 above)
Comment (kmova) [reply docs/releases.md:19]: timeout is valid.

File: hack/cspell-ignore.txt (hunk 8 above)
Comment (kmova) [reply hack/cspell-ignore.txt:10]: all seem to be used inter-changebly. but good idea to follow a convention.

File: hack/cspell-words.txt (hunk 9 above)
Comment (kmova) [reply hack/cspell-words.txt:396]: sharded .. from sharding

File: docs/releases.md (hunk 6 above)
Comment (ranjithwingrider) [reply docs/releases.md]: There is one time out. This can be timeout then...

**Review by ranjithwingrider — APPROVED** [review]
All changes are verified.  Please go ahead.

**Pull request merged**